habitat/tasks/nav/nav_task.py

```diff
@@ -515,8 +543,21 @@ def update_metric(self, episode, action):
                 map_agent_x - (self._ind_x_min - self._grid_delta),
                 map_agent_y - (self._ind_y_min - self._grid_delta),
             ),
+            "agent_angle": self.get_polar_angle(),
```

Comment:
Better be specific here:
```suggestion
            "agent_polar_angle": self.get_polar_angle(),
```

Reply:
The method should be just `get_agent_angle()` as there isn't a difference between cartesian and polar for angles.  i.e. a polar_angle doesn't really make sense.  Maybe `agent_look_angle` makes the most sense?

Reply:
@erikwijmans name should encode that it's top down single angle.

Reply:
Discussed privately, will name `agent_angle` for now